src/main/views/terms-and-conditions.njk

```diff
@@ -0,0 +1,172 @@
+      </p>
+
+      <h2 class="heading-medium">{{ t('Information provided by this service') }}</h2>
+
+      <p>
+        {{ t('This service provides information to support you. HMCTS cannot give legal advice on individual cases.') }}
+      </p>
+
+      <p>
+        {{ t('You should answer the questions in the service based on your circumstances, '
+        + 'and seek legal advice if you need it.') }}
+      </p>
+
+      <h2 class="heading-medium">{{ t('Applicable law') }}</h2>
+
+      <p>{{ t('Your use of this service and any dispute arising from its use will be '
+        + 'governed by and construed in accordance with the laws of England and Wales, '
+        + 'including but not limited to the:') }}</p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('Computer Misuse Act 1990') }}</li>
+        <li>{{ t('Data Protection Act 1998') }}</li>
+        <li>{{ t('Mental Capacity Act 2005') }}</li>
+      </ul>
+      <h2 class="heading-medium">{{ t('Responsible use of this service') }}</h2>
+
+      <p>
+        {{ t('The service is designed for people who need to use it or by others '
+        + 'on their behalf and only with their consent.') }}
+      </p>
+
+      <p>
+        {{ t('There are risks in using a shared computer (for example in an internet café) to access this service. '
+        + 'It’s your responsibility to be aware of these risks and to avoid using any computer which may leave your '
+        + 'personal information accessible to others. You’re responsible if you choose to leave a computer unprotected '
+        + 'while using this service.') }}
+      </p>
+
+      <p>
+        {{ t('You must take your own precautions to ensure that the way you access '
+        + 'this service does not expose you to the risk of:') }}
+      </p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('viruses') }}</li>
+        <li>{{ t('malicious computer code') }}</li>
+        <li>{{ t('other forms of interference which may damage your computer system') }}</li>
+      </ul>
+
+      <p>
+        {{ t('You must not misuse this service by knowingly introducing viruses, trojans, worms, logic bombs '
+        + 'or other material which is malicious or technologically harmful. '
+        + 'You must not attempt to gain unauthorised access to this service, '
+        + 'the system on which it is stored or any server, computer or database connected to it. '
+        + 'You must not attack this site via a denial-of-service attack or a distributed denial-of-service attack.') }}
+      </p>
+
+      <p>
+        {{ t('This online service contains several free-text fields in which you will need to enter '
+        + 'certain types of information. You should not enter sensitive information into these free-text fields. '
+        + 'Sensitive information may include, but is not limited to, '
+        + 'details of religious beliefs and financial information. '
+        + 'Entering sensitive information is done so at your own risk. ') }}
```

Comment:
Remove whitespace at the end of sentence.

Reply:
The whitespace should be there to separate the sentences.